src-tauri/src/lib.rs

```diff
@@ -10,7 +11,21 @@ use tauri::{Emitter, Manager};
 
 use tauri_plugin_dialog::DialogExt;
 use tauri_plugin_shell::ShellExt;
+use tinymist_project::{CompileOnceArgs, EntryReader, LspUniverse, TaskInputs, WorldProvider};
+
+/// A global shared instance to process typst in a workspace.
+// todo: load compile arguments
+static VERSE: LazyLock<LspUniverse> = LazyLock::new(|| {
+    CompileOnceArgs {
+        root: Some(PathBuf::from(std::env::var("CARGO_MANIFEST_DIR").unwrap()).join("..")),
+        input: Some("../main.typ".to_string()),
+        ..CompileOnceArgs::default()
```

Comment:
We need to change this in the future - from my testing this limits the root directory to be the TyX directory for now, right?